rules/python_rules.build_defs

```diff
@@ -145,6 +148,12 @@ def python_binary(name:str, main:str, srcs:list=[], resources:list=[], out:str=N
         test_only=test_only,
     )
 
+    debug_cmd, debugger = _debug_cmd("./$OUT")
+    if debug_cmd:
+        if debugger == "debugpy":
+            cmd = cmd.replace(' --zip_safe', '')
```

Comment:
qq: why is this needed?

Reply:
debugpy was freaking during some file access of some some sort IIRC